File: CS/TempCollection.Module.Win/ViewController1.cs
using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Diagnostics;
using System.Text;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.Persistent.Base;

namespace TempCollection.Module.Win {
    public partial class ViewController1 : ViewController {
        public ViewController1() {
            InitializeComponent();
            RegisterActions(components);
            TargetViewId = "Order_DetailView";
        }
        protected override void OnActivated() {
            base.OnActivated();
            ObjectSpace.Committing += new EventHandler<CancelEventArgs>(ObjectSpace_Committing);
        }
        protected override void OnDeactivating() {
            base.OnDeactivating();
            ObjectSpace.Committing -= new EventHandler<CancelEventArgs>(ObjectSpace_Committing);
        }
        void ObjectSpace_Committing(object sender, CancelEventArgs e) {
            Order obj = (Order)View.CurrentObject;
            obj.UpdateGrouping();
        }
    }
}

File: VB/TempCollection.Module.Win/ViewController1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Text

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.Persistent.Base

Namespace TempCollection.Module.Win
	Partial Public Class ViewController1
		Inherits ViewController
		Public Sub New()
			InitializeComponent()
			RegisterActions(components)
			TargetViewId = "Order_DetailView"
		End Sub
		Protected Overrides Sub OnActivated()
			MyBase.OnActivated()
			AddHandler ObjectSpace.Committing, AddressOf ObjectSpace_Committing
		End Sub
		Protected Overrides Sub OnDeactivating()
			MyBase.OnDeactivating()
			RemoveHandler ObjectSpace.Committing, AddressOf ObjectSpace_Committing
		End Sub
		Private Sub ObjectSpace_Committing(ByVal sender As Object, ByVal e As CancelEventArgs)
			Dim obj As Order = CType(View.CurrentObject, Order)
			obj.UpdateGrouping()
		End Sub
	End Class
End Namespace
